README.md

```diff
@@ -3,8 +3,8 @@
 ---
 
 :warning: **Shippo no longer actively maintains this library** <br>
-However, you can use it as a starting point and update it for your needs and future changes to our APIs.<br>
-You can use our [Open API Specification](https://docs.goshippo.com/shippoapi/public-api/) to build your own library using tools like the [OpenAPI Generator](https://openapi-generator.tech/) and [Swagger Codegen](https://swagger.io/tools/swagger-codegen/).
+
+Use our latest [Shippo Python SDK](https://github.com/goshippo/shippo-python-sdk) 🚀

```

Comment:
This SDK is not maintained. It seems like a good idea to refer but I'm not sure it adds value in this case.

Reply:
Nvm, I just saw the new release.